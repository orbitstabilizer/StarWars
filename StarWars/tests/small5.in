1
Sector-1 REPUBLIC
10
Jedi jedi1 10000 10 10
Jedi jedi2 1000 10 10
Jedi jedi3 100 30 10
Jedi jedi4 10 10 10
Jedi jedi5 1000 10 10
Sith sith1 10000 100 10
Sith sith2 1000 100 10
Sith sith3 100 100 10
Sith sith4 10 0 110
Sith sith5 100 100 10
10
RepublicCruiser cruiser1 1 1
1 1
RepublicCruiser cruiser2 1 2
1 2
RepublicCruiser cruiser3 1 3
1 3
SeparatistDestroyer destroyer1 1 4
1 6
SeparatistDestroyer destroyer2 1 5
1 7
RepublicCruiser cruiser4 1 6
1 4
RepublicCruiser cruiser5 1 7
1 5
SeparatistDestroyer destroyer3 1 8
1 8
SeparatistDestroyer destroyer4 1 9
1 9
SeparatistDestroyer destroyer5 1 10
1 10
1
11 1