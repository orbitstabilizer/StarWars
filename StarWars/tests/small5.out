Warship cruiser1 in (Sector-1, 4)
jedi1 30723
Warship cruiser5 in (Sector-1, 8)
jedi5 4023
Warship cruiser2 in (Sector-1, 5)
jedi2 3723
Warship destroyer5 in (Sector-1, 10)
sith5 1302
Warship cruiser3 in (Sector-1, 3)
jedi3 1203
Warship destroyer4 in (Sector-1, 9)
sith4 542
Warship destroyer1 is destroyed by cruiser1 in (Sector-1,4)
Warship destroyer2 is destroyed by cruiser2 in (Sector-1,5)
Warship cruiser4 is destroyed by destroyer4 in (Sector-1,9)
Warship destroyer3 is destroyed by cruiser5 in (Sector-1,8)
Sith sith1 is free
10410
Jedi jedi1 is in cruiser1
10040
Sith sith2 is free
1410
Jedi jedi5 is in cruiser5
1140
Jedi jedi2 is in cruiser2
1040
Sith sith3 is killed by jedi5
510
Sith sith5 is in destroyer5
510
Jedi jedi3 is in cruiser3
200
Sith sith4 is in destroyer4
130
Jedi jedi4 is killed by sith4
-50
